package.json

```diff
@@ -21,6 +21,7 @@
   },
   "devDependencies": {
     "servertest": "^1.2.1",
+    "standard": "^10.0.2",
```

Comment:
This PR should only do a single thing -- adding standard should be in a separate PR.